src/box/lua/net_box.lua

```diff
@@ -759,6 +772,21 @@ function remote_methods:_request(method, opts, format, stream_id, ...)
         timeout = deadline and max(0, deadline - fiber_clock())
         self:wait_state('active', timeout)
     end
+
+    -- Ping is the only one, who need an error to be returned
+    -- So let's not change _request's interface only for that
+    return res, err
+end
+
+--
+-- Behave like _request_ex but throw an exception in case of connection
+-- related errors too
```

Comment:
That is the contrary of what ex means. `ex` means exception. Functions ending with `ex` or `xc` (most likely we should use `xc` - this is used in C code and in vshard code too) declare that they throw exceptions. Here you stated the contrary.

Reply:
Just thought that `ex` can be used as a synonym of former or old. Don't really want to change the external behavior of `_request`'s error handling as it's used in dozens of tests. We can take out the logic of `_request` in another function, which will be wrapped in `_request`. Maybe there's the proper name in tarantool for this internal function too?